spec/observables/from-spec.ts

```diff
@@ -118,5 +119,26 @@ describe('from', () => {
         );
       expect(nextInvoked).to.equal(false);
     });
+    it(`should accept a function`, (done) => {
```

Comment:
`from` should not accept a function.

`from((...args) => subject.next(...args))` should not work and should error.

The change to `isArrayLike` I can understand, I'm uncertain about this test.

Reply:
if `from` accepts a `Promise`, shouldn't it be able to accept a `handler`?

Reply:
I see, the function has `Symbol.observable` on it. I missed that. Apologies.